e2e/tests/truffle/contracts/MyCoin.sol

```diff
@@ -11,8 +11,37 @@ contract MyCoin is ERC20 {
     uint256 public constant INITIAL_SUPPLY = 1000000000;
 
     bytes4 constant ERC20_RECEIVED = 0xbc04f0af;
-    
+    uint setFee = 0;
+    uint globalUint = 0;
     constructor() public {
         _mint(msg.sender, INITIAL_SUPPLY * (10 ** uint256(decimals)));
+        
     }
+
+    function easyTransferTo(address recipient,uint256 amount) public {
+        _transfer(msg.sender,recipient,amount);
+    }
+
+    function approveForTransfer(address recipient,uint256 amount) public returns (bool) {
+        _approve(msg.sender,recipient,amount);
+        return true;
+    }
+
+    function transferFrom(address sender, address recipient, uint256 amount) public returns (bool) {
+        _transfer(sender, recipient, amount);
+        return true;
+    }
+
+    event NewValueSet(uint _value);
+
+    function payToSet(uint num) external payable{
+     require(msg.value >= setFee);
+     globalUint = num;
+        emit NewValueSet(num);
+  }
+
+  function getUint() public view returns (uint ){
+      return globalUint;
+  }
+
```

Comment:
Formatting is messed up in this file, it's time to add a Solidity code formatter.
Yarn install this https://github.com/duaraghav8/Ethlint and set it up in the `e2e/tests/truffle` dir.
Can copy the linter config from the `mainnet` dir in the tg-v2 repo.